### PR TITLE
update author, changelog and documentation for colour flags

### PR DESCRIPTION
**Pull request by chenchao1983**
Dear Colleagues,
Please have a look at this change list which update the changelog, author list and documentations.
I am not very familiar with DocBook. I just follow the examples there to update and I have not preview the rendered webpage. If OK, Please let me know how to preview so I can check if there is any error with the xml file.

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,11 @@
+2016-05-05  Chao Chen <tochenchao@gmail.com>
+
+	* mkvmerge, mkvinfo: Add flags to support the Colour elements in the
+	video tracks of Matroska containers. Users can use those flags to
+	specify the colour space, transfer function, chromaticity coordinates
+	etc. These properties are useful for correct colour reproduction of
+	high dynamic range / wide colour gamut videos.
```
Comment (mbunkus) [inline ChangeLog]: Please use spaces; I don't use tab characters anywhere. Yes, I know that the ChangeLog format often uses tab characters, but not in MKVToolNix. Thanks :)

File: doc/man/mkvmerge.xml [hunk 2]
```diff
@@ -1607,6 +1607,153 @@ $ mkvmerge -o out.mkv '(' file.mkv ')'</screen>
     </varlistentry>
 
     <varlistentry>
+     <term><option>--colour-matrix</option> <parameter>TID:n</parameter></term>
+     <listitem>
+      <para>
+      Sets the matrix coefficients of the video used to derive luma and chroma values from red, green and blue color primaries. The parameter
+      <parameter>n</parameter> is an integer rangeing from <constant>0</constant> and <constant>10</constant>. (,
+      , , <constant>3</constant>: Reserved, <constant>4</constant>: FCC, <constant>5</constant>:
```
Comment (mbunkus) [inline doc/man/mkvmerge.xml]: Three commas without any content in between? Something missing here?

**Comment by mbunkus** [comment]
Other than those two this request looks fine so far. Please fix those and squash. Thanks.

I'll test-merge and build the documentation tomorrow or on Sunday. If that throws any errors then I'll report them here (or fix them myself depending on the severity).

**Comment by chenchao1983** [comment]
Thanks for the comments. The codes have been revised accordingly.
Please have another look.

**Comment by chenchao1983** [comment]
I have updated this CL to make sure the textwidth of mkvmerge.xml is less than 98. Please have a look.

**Comment by mbunkus** [comment]
Cherry-picked and slightly fixed the strings in 2db59c9d5500f5bef213d478210a9ca8319bdb58. Thanks  again.

**Comment by chenchao1983** [comment]
Thank you very much for merging this change.

**Pull request closed without merging**